javascript/src/main/components/Nav/AppNavbar.js

```diff
@@ -39,6 +40,7 @@ function AppNavbar() {
           { (isMember || isAdmin)  &&
               <NavDropdown title="Search">
                   <NavDropdown.Item href="/member/messages/search">Message Search</NavDropdown.Item>
+                  <NavDropdown.Item href="/admin/students/team1">Manage Team1 Students</NavDropdown.Item>
```

Comment:
I'm confused by both the url and the UX choice here.   
* Why is this `team1` and not `team`?
* Why is this `Manage Team1 Students` and not `Manage Team Students`

Reply:
thanks for pointing it out, it was originally an idea we had to manage each team, but never got around to fully implementing. removed from the code now since it is not used